uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-widget/leave-widget-absence-types-amount-taken.component.spec.js

```diff
@@ -5,8 +5,11 @@ define([
   'common/moment',
   'mocks/helpers/controller-on-changes',
   'mocks/data/option-group.data',
+  'mocks/apis/absence-period-api-mock',
+  'mocks/apis/absence-type-api-mock',
   'mocks/apis/leave-request-api-mock',
-  'leave-absences/shared/components/leave-widget/leave-widget-absence-types-amount-taken.component'
```

Comment:
why are you not using this component anymore?

Reply:
the `leave-widget.component` is including it since that's the base module.